### ISSUE TITLE
Pressing SUPER key on keyboard should open Steam menu

### ISSUE THREAD

**Issue by japawBlob** [ValveSoftware/SteamOS#1361]
#### Your system information

* Steam client version: 1705108172
* SteamOS version: 3.5.7, build 20231122.1
* Opted into Steam client beta?: No
* Opted into SteamOS beta?: No 
* Have you checked for updates in Settings > System?: Yes

#### Please describe your issue in as much detail as possible:
When in big picture mode, the SUPER key (or Windows key) should open the Steam menu, like when pressing STEAM button on the Steam deck.

I get the paradox of having big-picture open on the desktop and then deciding if the super key should interact with the big picture, or the desktop (honestly not even sure if it can be overrided)
But when running bigpicture without background desktop, I believe the behaviour should be as described.

#### Steps for reproducing this issue:

1. Start Steam deck
2. Connect keyboard
3. press super

**Comment by dg10mcdos** [comment]
Would give an arm and leg for this

**Comment by Bondarenko199** [comment]
True. There should be at least one way to open menu or at least to switch between home screen and the game using a keyboard

**Comment by wh0am1-dev** [comment]
Agree, this should already be implemented.

**Comment by oSoMoN** [comment]
There's already a couple of keyboard shortcuts:
 * `Ctrl+Alt+1` : open Steam menu
 * `Ctrl+Alt+2` : open the Quick Settings menu

People have expressed in other issues that these key combinations might not be ideal as they could potentially conflict with in-game shortcuts, but I think that's a separate issue to what you're reporting, right?

**Comment by wh0am1-dev** [comment]
@oSoMoN those shortcuts only work while in steamOS, once you open a game they don't work. There are other ways to open the steam and "..." menus while in steamOS: clicking in the steam button bottom-left with a mouse, clicking the notification icon top-right (or focusing it with the keyboard arrows and pressing enter). So even have alternatives for those shortcuts in steamOS. But within a game there's simply no way of opening those menus with just a keyboard and a mouse, you need a controller.

I reiterate myself, this should already be implemented. I think it's not a crazy idea to bind the steam menu to the super (win) key in the keyboard. I don't mind if there's no shortcut for the "..." menu. If we can open the steam menu, we can hack our ways into the other menu through notifications or any other way. But there's nothing we can use.

**Comment by Electronicks** [comment]
The shortcut to open the steam menu can be rebound, but not to SUPER.

Also, there usually is a left and right SUPER key. They could be distinguished to open the steam menu on the left and QAM on the right.

This can be in gaming mode only.
